core_contracts/governance/governance.py

```diff
@@ -58,6 +64,83 @@ def getVotersCount(self, name: str) -> dict:
         vote_index = ProposalDB.proposal_id(name, self.db)
         proposal = ProposalDB(var_key=vote_index, db=self.db)
         return {'for_voters': proposal.for_voters_count.get(), 'against_voters': proposal.against_voters_count.get()}
+    
+    @external
+    @only_owner
+    def setMinimumVoteDuration(self, duration: int) -> None:
+        """
+        Set the minimum vote duration.
```

Comment:
We have been trying to follow the docstring convention used in most of the ICON code, an example of which is here - https://github.com/icon-project/icon-service/blob/master/iconservice/iconscore/icon_score_base.py#L649

Single line descriptions can also go on the same line as the opening triple quote, but that is a convention that they seem to go by about 50% of the time. Not sure offhand if it is preferred in Python style in general. Python style guide give a spec for multiline docstrings here - https://www.python.org/dev/peps/pep-0257/#id17 I'd refer to that in general, but also simply opt for consistency throughout.

Reply:
Sounds good. I'll make changes to fit that convention.